### PR TITLE
Add Giter8 template usage in README

### PR DESCRIPTION
**Pull request by JesusMtnez**
This PR adds documentation in README to start a new project using Giter8 template for `twitter4s`

---
PR from Lambda World 2017 Scala Center Spree 😄

### CODE REVIEW

**Comment by DanielaSfregola** [comment]
This completes #143

File: README.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ Also, you need to include the library as your dependency:
 libraryDependencies += "com.danielasfregola" %% "twitter4s" % "5.1"
 ```
 
+Setup using Giter8
```
Comment (DanielaSfregola) [inline README.md]: Can we call this section just "Giter8"?

File: README.md [hunk 2]
```diff
@@ -31,6 +31,15 @@ Also, you need to include the library as your dependency:
...
+------------------
+
+If you are starting from scratch, you can use [`twitter4s.g8`](https://github.com/DanielaSfregola/twitter4s.g8) template to start your project. This template contains examples for both REST and streaming client.
```
Comment (DanielaSfregola) [inline README.md]: Please change "streaming" to "Streaming" for consistency with the rest of the file

File: README.md (hunk 1 above)
Comment (JesusMtnez) [reply README.md]: Yeah, of course

File: README.md (hunk 2 above)
Comment (JesusMtnez) [reply README.md]: Totally agree

**Pull request merged**